types/mercadopago-sdk-js/modules/bricks.d.ts

```diff
@@ -83,8 +83,11 @@ declare namespace bricks {
   enum WalletButtonValueProp {
     PRACTICALITY = "practicality",
     CONVENIENCE = "convenience",
+    CONVENIENCE_ALL = "convenience_all",
     SECURITY_DETAILS = "security_details",
     SECURITY_SAFETY = "security_safety",
+    CREDITS= "convenience_credits",
+    SMART= "smart_option",
```

Comment:
Não seria melhor deixar o nome da chave igual ao do valor da value prop?
```suggestion
    CONVENIENCE_CREDITS = "convenience_credits",
    SMART_OPTION = "smart_option",
```